integration-tests/src/test/groovy/com/okta/sdk/tests/it/UsersIT.groovy

```diff
@@ -306,10 +306,13 @@ class UsersIT extends ITSupport implements CrudTestSupport {
         // 3. List roles for the user and verify added role
         assertPresent(user.listAssignedRoles(), role)
 
-        // 4. Remove role for the user
+        // 4. Verify added role
```

Comment:
This has the potential to be a flaky test? Worth adding a sleep before this assertion?